mlflow/utils/databricks_utils.py

```diff
@@ -98,6 +116,11 @@ def get_notebook_path():
     return _get_extra_context("notebook_path")
 
 
+def get_cluster_id():
+    spark_session = get_active_spark_session()
```

Comment:
Add check: if `spark_session` is `None`, should return `None` here to be safe, rather than trying to access `spark_session.conf`

Reply:
Original thought is that it might be redundant as we will do "is_in_cluster_check" first, which has the logic that returns false if spark_session is none, or spark_session exists while spark_session.conf is none.
But as a standalone function, we should add that check as you mentioned